### PR TITLE
feat (UriToBitmap) add test

### PR DESCRIPTION
**Pull request by giovanni-nic-kuama**
fix (FindPaperSheetContoursInstrumentedTes) assertions needed to be changed

fix (PerspectiveTransform) change corner orders to match the correct corner order

address [ADS-4] Test UriToBitmap

### CODE REVIEW

File: app/src/androidTest/java/net/kuama/documentscanner/domain/FindPaperSheetContoursInstrumentedTest.kt [hunk 1]
```diff
@@ -31,20 +31,20 @@ class FindPaperSheetContoursInstrumentedTest {
                     if (resultingCorners != null) {
```
Comment (matteoKuama) [inline app/src/androidTest/java/net/kuama/documentscanner/domain/FindPaperSheetContoursInstrumentedTest.kt:31]:
```suggestion
    fun findPaperSheetUseCaseValidInputReturnCorrectResult() {
        TestUtils.loadOpenCvLibrary()
        val inputStream: InputStream? = this.javaClass.classLoader?.getResourceAsStream("document-picture.jpg")
        val bitmap = BitmapFactory.decodeStream(inputStream)
        runBlocking {
            findPaperSheetUseCase(
                FindPaperSheetContours.Params(bitmap)
            ) { either ->
                either.mapRight { resultingCorners ->
                    if (resultingCorners != null) {
```

Comment (matteoKuama) [reply app/src/androidTest/java/net/kuama/documentscanner/domain/FindPaperSheetContoursInstrumentedTest.kt:31]: there is a way to write the name of test functions as free text between two ' '

File: app/src/main/java/net/kuama/documentscanner/domain/PerspectiveTransform.kt [hunk 2]
```diff
@@ -40,10 +40,9 @@ class PerspectiveTransform : UseCase<Bitmap, PerspectiveTransform.Params>() {
 
         val transformedDocumentMatrix = Mat(maxHeight.toInt(), maxWidth.toInt(), CvType.CV_8UC4)
 
-        srcMat.put(0, 0, params.corners.topLeft.x, params.corners.topLeft.y,
-            params.corners.topRight.x, params.corners.topRight.y, params.corners.bottomRight.x,
-            params.corners.bottomRight.y, params.corners.bottomLeft.x, params.corners.bottomLeft.y
-        )
+        srcMat.put(0, 0, params.corners.bottomLeft.x, params.corners.bottomLeft.y,
```
Comment (matteoKuama) [inline app/src/main/java/net/kuama/documentscanner/domain/PerspectiveTransform.kt]: format
```suggestion
        srcMat.put(0,
         0, 
         params.corners.bottomLeft.x, 
         params.corners.bottomLeft.y,
```

**Review by matteoKuama — CHANGES_REQUESTED** [review]

File: app/src/androidTest/java/net/kuama/documentscanner/domain/FindPaperSheetContoursInstrumentedTest.kt (hunk 1 above)
Comment (GabM3) [reply app/src/androidTest/java/net/kuama/documentscanner/domain/FindPaperSheetContoursInstrumentedTest.kt:31]: also use snake case

Comment (matteoKuama) [reply app/src/androidTest/java/net/kuama/documentscanner/domain/FindPaperSheetContoursInstrumentedTest.kt:31]: https://kotlinlang.org/docs/coding-conventions.html#names-for-test-methods

**Review by matteoKuama — DISMISSED** [review]

**Review by SimoneBressan — DISMISSED** [review]

**Review by SimoneBressan — APPROVED** [review]

**Pull request merged**